app/models/pglogical_subscription.rb

```diff
@@ -34,6 +34,7 @@ def self.find_by_id(to_find)
 
   def save!
     assert_valid_schemas!
+    assert_different_region!
```

Comment:
Personally, I think we should check the different region first.  No point in checking the schema (more expensive) if the region check fails (less expensive)

Reply:
Good point, I'll change.